main.go

```diff
@@ -166,6 +215,15 @@ func main() {
 		setupLog.Error(err, "problem running manager")
 		os.Exit(1)
 	}
+
+	// create cred request
+	c := mgr.GetClient()
+	if err := c.Create(context.TODO(), credReq); err != nil {
```

Comment:
Is it potentially the case this does not exist as an API for specific installs of openshift?

I am thinking about some of the edge scenarios, where we have way fewer operators installed AFAICT.

Reply:
Done